### ISSUE TITLE
lsp-ui-sideline not working when update to latest version

### ISSUE THREAD

**Issue by zy9306** [emacs-lsp/lsp-ui#472]
I download the lsp-start-plain from https://raw.githubusercontent.com/emacs-lsp/lsp-mode/master/scripts/lsp-start-plain.el and add my config at the last，then open a py file, and not show sideline.

The previous version is ok, but I’m not sure which version...
```
(setq lsp-ui-sideline-enable t
      lsp-ui-sideline-show-symbol t
      lsp-ui-sideline-show-hover t
      lsp-ui-sideline-show-flycheck t
      lsp-ui-sideline-show-code-actions nil
      lsp-ui-sideline-show-diagnostics nil)

(setq lsp-ui-doc-enable nil)
(setq lsp-ui-imenu-enable nil)
(setq lsp-ui-peek-enable nil)
```

**Comment by yyoncho** [comment]
which python language server do you use?

**Comment by zy9306** [comment]
@yyoncho mspyls with lsp-python-ms, the server has not been updated. This problem occurred after an update of lsp-mode, but I am not sure which version

**Comment by yyoncho** [comment]
I know why this has happened, just wanted to know which server to test with.

**Comment by kurnevsky** [comment]
Was fixed in https://github.com/emacs-lsp/lsp-ui/pull/466 too...

**Comment by yyoncho** [comment]
> Was fixed in #466 too...

I had a Deja view when I was doing the fix...
